test/integration/cart_integration_test.rb

```diff
@@ -64,35 +69,35 @@ class CartIntegrationTest < ActionDispatch::IntegrationTest
     click_link_or_button('Cart')
 
     within('#cart_items') do
-      assert page.has_content?('coffee')
+      assert page.has_content?('produce only the finest')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.